### PR TITLE
fix/rename segmentation namespace

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Description

- rename segmentation directory
- update topic namespace `/perception/object_segmentation/pointcloud` -> `/perception/obstacle_segmentation/pointcloud`
- simulation stack is fixed in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/739
- launcher is fixed in https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/139
- aip_launcher is fixed in https://github.com/tier4/AutowareArchitectureProposal_aip_launcher/pull/6

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Read [pull request guidelines][pull-request-guidelines]
- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR / build-and-test-pr**: Required to pass before the merge.
- **Build and test for PR / clang-tidy-pr**: NOT required to pass before the merge. It is up to the reviewer(s). Found false positives? See the [guidelines][clang-tidy-guidelines].
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[clang-tidy-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/ClangTidyGuideline/
[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[pull-request-guidelines]: https://tier4.github.io/autoware.proj/tree/main/developer_guide/PullRequestGuideline/
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

**Comment by tkimura4** [comment]
`/perception/object_segmentation/pointcloud` remains.

https://github.com/tier4/AutowareArchitectureProposal.iv/blob/14bff6670fd205568ece869fcc244c239962826e/simulator/util/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

**Comment by satoshi-ota** [comment]
> `/perception/object_segmentation/pointcloud` remains.
> 
> https://github.com/tier4/AutowareArchitectureProposal.iv/blob/14bff6670fd205568ece869fcc244c239962826e/simulator/util/dummy_perception_publisher/launch/dummy_perception_publisher.launch.xml

@tkimura4 Thank you for your review :+1: and, `dummy_perception_publisher` is fixed in the other PR (https://github.com/tier4/AutowareArchitectureProposal.iv/pull/739)

**Comment by tkimura4** [comment]
https://github.com/tier4/AutowareArchitectureProposal_launcher/pull/139
と一緒にlogging_simualtor.launch.xmlでチェック. 特に問題なさそう.

image

ros2 topic info /perception/obstacle_segmentation/pointcloud -v
```
Type: sensor_msgs/msg/PointCloud2

Publisher count: 1

Node name: occupancy_grid_map_outlier_filter
Node namespace: /perception/obstacle_segmentation/ground_segmentation
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: PUBLISHER
GID: b6.f0.10.01.db.5f.2b.bb.4b.96.8c.da.00.00.3d.03.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Subscription count: 7

Node name: surround_obstacle_checker
Node namespace: /planning/scenario_planning/lane_driving/motion_planning
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: 18.08.10.01.fa.03.24.af.98.f2.b7.50.00.00.28.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: obstacle_stop_planner
Node namespace: /planning/scenario_planning/lane_driving/motion_planning
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: 18.08.10.01.fa.03.24.af.98.f2.b7.50.00.00.67.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: rviz2
Node namespace: /
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: 28.d4.10.01.cd.a9.ba.a5.0e.a5.7e.a1.00.00.67.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: voxel_grid_filter_dpc1912002_ubuntu20_474722_7727678858549655902
Node namespace: /perception/object_recognition/detection/euclidean/clustering
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: 5a.44.10.01.aa.ac.58.58.68.4d.27.10.00.00.36.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: behavior_velocity_planner
Node namespace: /planning/scenario_planning/lane_driving/behavior_planning
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: 9e.ed.10.01.2d.e3.a5.86.10.96.66.9c.00.00.4a.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: costmap_generator
Node namespace: /planning/scenario_planning/parking
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: a2.b6.10.01.b1.26.d5.e6.cb.27.df.c1.00.00.24.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds

Node name: autoware_state_monitor
Node namespace: /system
Topic type: sensor_msgs/msg/PointCloud2
Endpoint type: SUBSCRIPTION
GID: b1.35.10.01.18.d3.47.07.b8.1e.81.fa.00.00.1d.04.00.00.00.00.00.00.00.00
QoS profile:
  Reliability: BEST_EFFORT
  Durability: VOLATILE
  Lifespan: 9223372036854775807 nanoseconds
  Deadline: 9223372036854775807 nanoseconds
  Liveliness: AUTOMATIC
  Liveliness lease duration: 9223372036854775807 nanoseconds
```

File: planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/README.md [hunk 1]
```diff
@@ -47,14 +47,14 @@ As mentioned in stop condition section, it prevents chattering by changing thres
...
-| `/perception/object_segmentation/pointcloud` | `sensor_msgs::msg::PointCloud2`                        | No ground pointcloud |
-| `/perception/object_recognition/objects`     | `autoware_auto_perception_msgs::msg::PredictedObjects` | Dynamic objects      |
-| `/localization/kinematic_state`              | `nav_msgs::msg::Odometry`                              | Current twist        |
-| `/tf`                                        | `tf2_msgs::msg::TFMessage`                             | TF                   |
-| `/tf_static`                                 | `tf2_msgs::msg::TFMessage`                             | TF static            |
+| Name                                           | Type                                                   | Description          |
+| ---------------------------------------------- | ------------------------------------------------------ | -------------------- |
+| `~/input/trajectory`                           | `autoware_auto_planning_msgs::msg::Trajectory`         | Reference trajectory |
+| `/perception/obstacle_segmentation/pointcloud` | `sensor_msgs::msg::PointCloud2`                        | No ground pointcloud |
```
Comment (yukkysaito) [inline planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/README.md:53]: No ground pointcloudよりは
Pointcloud of obstacles that the ego-vehicle must not hitとかPointcloud of obstacles where the ego-vehicle should stopみたいなニュアンスのほうが良いかもです

File: control/obstacle_collision_checker/README.md [hunk 2]
```diff
@@ -57,13 +57,13 @@ If any collision is found on predicted path, this module sets `ERROR` level as d
...
-| `~/input/trajectory`                         | `autoware_auto_planning_msgs::msg::Trajectory` | Predicted trajectory |
-| `/perception/object_segmentation/pointcloud` | `sensor_msgs::msg::PointCloud2`                | No ground pointcloud |
-| `/tf`                                        | `tf2_msgs::msg::TFMessage`                     | TF                   |
-| `/tf_static`                                 | `tf2_msgs::msg::TFMessage`                     | TF static            |
+| Name                                           | Type                                           | Description          |
+| ---------------------------------------------- | ---------------------------------------------- | -------------------- |
+| `~/input/trajectory`                           | `autoware_auto_planning_msgs::msg::Trajectory` | Reference trajectory |
+| `~/input/trajectory`                           | `autoware_auto_planning_msgs::msg::Trajectory` | Predicted trajectory |
+| `/perception/obstacle_segmentation/pointcloud` | `sensor_msgs::msg::PointCloud2`                | No ground pointcloud |
```
Comment (yukkysaito) [inline control/obstacle_collision_checker/README.md:64]: 同様
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/742/files#r758039152

File: planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/README.md (hunk 1 above)
Comment (satoshi-ota) [reply planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/README.md]: fix in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/742/commits/285f648deed54f17f1a5f8c5d0aac1f8161a9fb6

File: control/obstacle_collision_checker/README.md (hunk 2 above)
Comment (satoshi-ota) [reply control/obstacle_collision_checker/README.md]: fix in https://github.com/tier4/AutowareArchitectureProposal.iv/pull/742/commits/285f648deed54f17f1a5f8c5d0aac1f8161a9fb6

**Review by yukkysaito — APPROVED** [review]

**Pull request merged**